service/src/main/java/com/commercetools/pspadapter/payone/mapping/MappingUtil.java

```diff
@@ -12,16 +12,23 @@
 import io.sphere.sdk.models.Reference;
 import io.sphere.sdk.payments.Payment;
 import io.sphere.sdk.types.CustomFields;
+import org.apache.commons.lang3.StringUtils;
+import org.javamoney.moneta.function.MonetaryUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
+import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
-import java.util.Locale;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
It's common pattern to squeeze imports to wildcard if there are more than N imports of the same package. The same opinion in [Clean Code](https://stackoverflow.com/a/1992695/907576). The same opinion in default _Intellij Idea_ settings, the same opinion i have :)